src/blockdom/markup.ts

```diff
@@ -0,0 +1,31 @@
+import { html } from "./html";
```

Comment:
this file should not be in blockdom.it should be at the root /

Reply:
hmm, actually, the `markup` function should be in `utils` and the `safeout` code should be in `app/template_helpers`